src/components/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+interface InputProps {
+    onClick?: () => void;
+    className?: string;
+    title: string;
+  }
+
+const Input=({
+    className = '',
+    title,
+  }: InputProps)=> {
+  const [isChecked, setIsChecked] = useState(false);
```

Comment:
As per our [React Guideline](https://pj100.esa.io/posts/16722#%E3%82%A2%E3%83%B3%E3%83%81%E3%83%91%E3%82%BF%E3%83%BC%E3%83%B3%E3%81%AE%E6%94%B9%E5%96%84), useState should have predefined type like so:

```suggestion
  const [isChecked, setIsChecked] = useState<boolean>(false);
```

Reply:
Changed, will take take care from next time !